app/scripts/TiledPlot.js

```diff
@@ -795,6 +795,14 @@ class TiledPlot extends React.Component {
     TRACK_LOCATIONS.forEach(location => {
       if (tracks[location]) {
         tracks[location].forEach(track => {
+          if (track.contents) {
+            for (let i = 0; i < track.contents.length; i++) {
+              track.contents[i].position = location;
+            }
```

Comment:
Syntax suggestion:

```javascript
track.contents.forEach(content => { content.position = location; });
```

as it's more declarative rather than imperative.